pkg/framework/guest.go

```diff
@@ -27,16 +24,19 @@ const (
 type GuestConfig struct {
 	Logger micrologger.Logger
 
+	HostK8sClient kubernetes.Interface
```

Comment:
We don't have validation for this config in `NewGuest`

Reply:
Fixed.